views/layout.erb

```diff
@@ -118,9 +118,15 @@
 
 <div class="container" role="main">
   <%= yield %>
-
+  <!-- error message  -->
+  <% if @error_message %>
+    <% @error_message.each do |msg| %>
+        <div class='row'><div class='alert alert-warning' role='alert'><%= msg %></div></div>
+    <% end %>
+ <% end %><!-- /.error message  -->
```

Comment:
You can coerce to array to avoid need for if statement:

```
<% Array(@error_message).each do |msg| %>
```

thats because `Array(nil) #=> []` and `Array([1,2,3]) # =>  [1,2,3]`

Reply:
And `@some_instance_variable_that_does_not_exist` will resolve to nil:

```
irb(main):004:0> class Foo
irb(main):005:1> def bar
irb(main):006:2> Array(@one)
irb(main):007:2> end
irb(main):008:1> end
=> :bar
irb(main):010:0> Foo.new.bar
=> []
```

Reply:
I changed if statement to `<% Array(@error_messages).each do |msg| %>` and solved small code style issues you mentioned. Tested and all good.